### PR TITLE
allow to show only summary in text output

### PR DESCRIPTION
**Pull request by unixslayer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/.idea/
```
Comment (jaylinski) [inline .gitignore:1]: Please remove. (You can add this to your global gitignore)

**Comment by jaylinski** [comment]
@unixslayer What's your use case for this change?

**Comment by unixslayer** [comment]
@jaylinski For me, this is rather cosmetic change. Most of the time I want to see summary instead of text formatter to show coverage for each class separately, especially when I have a lot of them.

**Comment by shulard** [comment]
Hello,

Thank you for the PR, this seems a nice tiny addition to help use reports for more people.

**Comment by shulard** [comment]
Thank you @unixslayer  !

**Comment by TiMESPLiNTER** [comment]
When is a new release planned that includes this new option?

**Comment by jaylinski** [comment]
@TiMESPLiNTER See #32.

**Pull request merged**